cartographer_ros_msgs/package.xml

```diff
@@ -33,4 +33,6 @@
   <depend>geometry_msgs</depend>
 
   <build_depend>message_generation</build_depend>
+  
+  <exec_depend>message_runtime</exec_depend>
```

Comment:
run_depend?

Reply:
As this package is [using package format 2](https://github.com/googlecartographer/cartographer_ros/blob/e1774c82a9e1a2ac24b9d2b5d853eae4aa828a3c/cartographer_ros_msgs/package.xml#L18) it should be [exec_depend](http://www.ros.org/reps/rep-0140.html#exec-depend-multiple) ad the `run_depend` tag doesn't exist in that format (it has been split into `exec_depend` and `build_export_depend`)